docrepr/webview.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*
+
+"""
+View object documentation HTML in a separate GUI window.
+
+Requires PyQt4 or PySide. Original code by
+anatoly techtonik <techtonik@gmail.com> is placed in
+public domain.
```

Comment:
It is not a difference license, but a note that original code submitted by me is placed into public domain and doesn't bear additional restrictions of Spyder license in case somebody will need to reuse it in their own project.